src/components/Menu/MenuBar.vue

```diff
@@ -161,14 +161,20 @@ export default {
 			return list
 		},
 		hiddenEntries() {
+			const entries = this.entries.filter(({ priority }) => {
+				// reverse logic from visibleEntries
+				return priority !== undefined && priority > this.iconsLimit
+			}).reduce((acc, entry) => {
+				// merge all entries children into the entries list
```

Comment:
```suggestion
				// If entry has children, merge them into list. Otherwise keep entry itself.
```

I didn't get what the comment said straight away, so here's a suggestion for being more explicit.